### ISSUE TITLE
MKS Robin Nano V 1.3, no firmware is loadedПроизводители ПИДАРАСЫ

### ISSUE THREAD

**Issue by ooomakon** [makerbase-mks/MKS-Robin-Nano-S-Firmware#5]
Hello, there is a MKS board Robin Nano V 1.3 STM32F407VET6, no firmware is loaded.Flash drives changed 5pcs.2 boards behave the same.
Bootloader flashed, did not help
https://github.com/makerbase-mks/MKS-Robin-Nano-V1.X/tree/master/Bootloader
https://github.com/makerbase-mks/MKS-Robin-Nano-S-Firmware/issues/3
![photo_2022-03-16_17-43-52](https://user-images.githubusercontent.com/59833449/158782341-59421634-b8e1-4938-bb69-65242f255177.jpg)

**Comment by poon-holder** [comment]
This is causing me grey hairs!!! I have the nano 1.2 and .3. Both have done this while flashing klipper. They will however both flash back to original proprietary FW (two trees,Flsun). 
The Robin_nano35.bin changes to the correct .cur extension after stalling on "100%" and then frozen on "booting" once restarted. 
There should be no illumination to the screen after a successful klipper flash. It should be blank. Note that in klipper config docs the two trees and others sujest to invert pins to keep screen low after boot. 
I have been able to get the sapphire running, it randomly sucked the firmware and got past the 100% screen of death. but I do NOT know how or why. At first I suspected i used a better SD card but had literally spent 4 hours+ trying different card and compiler combinations. 
There is;
https://github.com/makerbase-mks/MKS-Robin-Nano-V1.X/issues/113
and
https://github.com/makerbase-mks/MKS-Robin-Nano-V1.X/issues/116

however they are somwhat prematurely closed as the Quirk remains a quirk and is not fully rectified or understood. Seeminly a few have success after a fresh boot loader burn but this cant be the issue if they will load their original FW and isn't a fix for all. 

My only advice is brute force, I literally compiled that 35.bin 60+ times, diligently wiping the drive each time and trying different drives. It was just a matter of luck it seems. Some claim the connected TFT is to blame (fix=disconnect it) but that does not seem to help all. 
I had swapped the gate over to USB power (for ease) and had been flashing by simply plugging in the USB b to boot. 
Other sugest Linux throws issues, im between ubuntu and big sur so will try and compile it solely on Mac and report if I have any success. 

Are you having this issue with a klipper install too, or when flashing marlin?

**Comment by ooomakon** [comment]
When installing Marlin

**Comment by ooomakon** [comment]
Пидорская поддержка,так и не ответили гандоны китайские, 2 платы с одинаковой болячкой,говноеды ебучие!

**Comment by KottenGH** [comment]
I'm struggling with the exact same thing now, both Marlin and Klipper. The only thing I can flash it with is from Two Trees.

**Comment by adamo000** [comment]
> I'm struggling with the exact same thing now, both Marlin and Klipper. The only thing I can flash it with is from Two Trees.

Have you make it work? I try to flash klipper on the STM32F407.. but still no luck (have no idea what options have to be used in make config)

**Comment by KottenGH** [comment]
Yes, never got Marlin to work but Klipper still works fine. Wish I remember the exact procedure though… for one thing, I installed the “Rpi OS lite” instead of the prebuilt Mainsail/Fluidd packages. That made the difference for me. Then you use Kiauh and get all the stuff you need installed. 

Then of course it’s about choosing the correct chipset etc I don’t remember that stuff. The display will show something like that though, as it won’t work. You can turn it off so it’s blank that’s all you can do as far as I know.
